eiffel-vocabulary/EiffelIssueVerifiedEvent.md

```diff
@@ -16,33 +16,27 @@
 --->
 
 # EiffelIssueVerifiedEvent (IV)
-The EiffelIssueVerifiedEvent declares that an issue, typically a requirement, has been verified by some means. It is different from [EiffelTestCaseFinishedEvent](./EiffelTestCaseFinishedEvent.md) in that multiple test case executions may serve as the basis for a single verification or, conversely, multiple issues may be verified based on a single test case execution.
+The EiffelIssueVerifiedEvent declares that an issue, typically a requirement, has been verified by some means. It is different from [EiffelTestCaseFinishedEvent](./EiffelTestCaseFinishedEvent.md) in that multiple test case executions may serve as the basis for a single verification or, conversely, multiple issues may be verified based on a single test case execution. Issues may only maintain one link of type ISSUE at a time.
```

Comment:
We don't want to say that `IssueVerified` can only reference one issue. In this paragraph we mention that "multiple issues may be verified based on a single test case execution." What we do want to say is something along the lines of "At least one link to an issue must be present, whether it be `"SUCCESSFUL_ISSUE"`, `"FAILED_ISSUE"`, or `"INCONCLUSIVE_ISSUE"`.